src/components/App.js

```diff
@@ -30,6 +31,10 @@ const App = () => {
     setInput(event.target.value)
   }
 
+  const deleteToDo = (id) => {
+    setTodos(todos.filter((todo) => todo.id != id))
```

Comment:
No es ideal poner funciones dentro de parámetros (a excepción de callbacks, y la mayoría de las veces ni así) porque dificultan la lectura. Por más que sea una línea extra, una variable suma mucho a la legibilidad, ya que no tenemos que estar leyendo la función para ver qué hace.

A la hora de comparar, usar siempre el comparador estricto `!==` ya que es más seguro.

```suggestion
    const updatedTodos = todos.filter((todo) => todo.id !== id) 
    setTodos(updatedTodos)
```